File: ejercicio6/empleado.php
<?php
class Empleado{

    public $nombre; //Nombre del empleado
    public $sueldo; //Sueldo del empleado

    public function mostrarDetalles(){ //Muestra los datos del empleado

        echo "Nombre: ".$this->nombre."\n".
        "Sueldo: ".$this->sueldo."\n";

    }
}

class Gerente extends Empleado{

    public $departamento; //Departamento del gerente

    public function mostrarDetalles(){ //Muestra los datos del gerente

        return "Nombre: ".$this->nombre."\n".
        "Sueldo: ".$this->sueldo."\n".
        "Departamento: ".$this->departamento."\n";

    }
}

$empleado = new Empleado();
$empleado->nombre = "Juan Perales";
$empleado->sueldo = 150.5;

$empleado->mostrarDetalles();

$gerente = new Gerente();
$gerente->nombre = "Agapito Morales";
$gerente->sueldo = 247.3;
$gerente->departamento = "Ventas";

echo $gerente->mostrarDetalles();

?>

File: Ejercicio6/EJ69/src/Empleado.java

public class Empleado implements Identificable {

	String nombre;
	int id;
	
	public Empleado(String nombre, int id) { //Constructor
		
		this.nombre = nombre;
		this.id = id;
	}
	
	public void mostrarIdentidad() { //Muestra los datos del empleado
		
		System.out.println("Nombre: " + nombre);
		System.out.println("ID: " + id);
	}
}
